neo/rawio/blackrockrawio.py

```diff
@@ -646,7 +646,7 @@ def _get_timestamp_slice(self, timestamp, seg_index, t_start, t_stop):
         if t_start is None:
             ind_start = None
         else:
-            ts = np.math.ceil(t_start * self.__nev_basic_header['timestamp_resolution'])
+            ts = int(np.ceil(t_start * self.__nev_basic_header['timestamp_resolution']))
```

Comment:
The deprecation warning suggests replacing `np.math` with the built-in Python `math` module, so I think this could just be `ts = math.ceil(...)` (with an import of `math` above), since `math.ceil` returns an integer.

Reply:
Done!